pkg/plugins/runtime/k8s/webhooks/testdata/validation/cli-user_create_dpp.input.yaml

```diff
@@ -8,6 +8,8 @@ metadata:
   creationTimestamp: null
   annotations:
     k8s.kuma.io/synced: "true"
+  labels:
```

Comment:
why did you add `kuma.io/zone` label here? does it work without the zone label?

Reply:
it does not work, without zone label now

Reply:
why? it's a breaking change then no? Can't we just add the `kuma.io/zone` label if it's missing?

Reply:
I've check again and it is not needed, I don't remember why I've added it